src/sidebar/components/UserMenu.tsx

```diff
@@ -56,7 +56,10 @@ function UserMenu({ frameSync, onLogout, settings }: UserMenuProps) {
   const isProfileEnabled = store.isFeatureEnabled('client_user_profile');
 
   const onSelectNotebook = () => {
-    frameSync.notifyHost('openNotebook', store.focusedGroupId());
+    const groupId = store.focusedGroupId();
+    if (groupId) {
+      frameSync.notifyHost('openNotebook', groupId);
+    }
```

Comment:
If "Open notebook" is clicked before any groups are loaded, nothing will happen. This is the same as before for the end user, but the difference is that we're explicitly handling this case.